rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingSpec.scala

```diff
@@ -52,11 +52,22 @@ class CostAccountingSpec extends FlatSpec with Matchers with PropertyChecks with
 
   val contracts = Table(
     ("contract", "expectedTotalCost"),
-    ("""@0!(2)""", 33L),
-    ("""@0!(2) | @1!(1)""", 69L),
-    ("""for(x <- @0){ Nil }""", 64L),
-    ("""for(x <- @0){ Nil } | @0!(2)""", 73L),
-    /*
+    ("""@0!(2)""", 97L),
```

Comment:
I also got this

```
[info] - should be repeatable *** FAILED *** (25 seconds, 323 milliseconds)               
[info]   TimeoutException was thrown during property evaluation. (CostAccountingSpec.scala:110)   
[info]     Message: Task.runSyncUnsafe(25 seconds)                                             
[info]     Occurred at table row 0 (zero based, not counting headings), which had values (                                                                                                                 
[info]       contract = @0!(2),                                               
[info]       expectedTotalCost = 97                                                                                                                                                                        
[info]     )       
```
:/
Not sure if this is related

Reply:
This seems to be the general stability problem with this test...

Reply:
Just noticed that this was a timeout, most probably due to a larger amount of tests executed in parallel. If this fails from time to time on my machine then it definitely is going to fail in CI sooner or later. Better increase the timeout just in case

Reply:
I tried locally with both bigger and smaller timeouts, without visible difference... I'm slightly against increasing the timeout, because most likely 25s per test case is far more than needed, and anything that hangs for 10s will most likely hang for 50s too - so it'd only make local hiccups more cumbersome. Unless you insist? :)

Reply:
I don't insist, I just think this is a wise thing to do. `checkRepeatableCost` executes many tasks in parallel so some of them might get starved for quite a long while. I don't mind deferring the timeout increase until (if at all) it has become a nuisance in CI